tests/testthat/test-utils.R

```diff
@@ -24,3 +26,24 @@ test_that("h_record_all_outputs works as expected with no removal list given for
   messages = c("O nearly done", "oh noo"))
   expect_identical(result, expected)
 })
+
+
+# trace of matrix ---
```

Comment:
```suggestion
# h_tr ----
```
I think we need four dashes so it shows up as a section in RStudio. Plus let's follow our convention here to use the tested function name